index.Rmd

```diff
@@ -17,11 +17,11 @@ knitr::opts_chunk$set(echo = TRUE)
 - Discussion
 
 <!-- This code is commented out -->
-<img src = "brain_art.jpg" class = "displayleft" alt = "My title"/>
 
-```{r eval = F, echo = F}
-<img src = "brain_art.jpg" class = "displayleft" alt = "My title"/>
-```
+Let's just add another kitten
```

Comment:
Agree. More kittens = good, and in fact perhaps they need a dedicated section in this code.  How about if we add a more descriptive title as well.

Reply:
OK, I'll work on that and get back to you.